pxr/imaging/hdx/shaders/renderPass.glslfx

```diff
@@ -176,6 +176,75 @@ vec4 IntToVec4(int id)
                 ((id >> 24) & 0xff) / 255.0);
 }
 
+// calculate an integer hash value based on integer key and offset
+// this is used for the pick buffer hash map
+int fnv1Hash(int key, int offset)
+{
+    const int prime = 16777619;
+
+    int ret = offset;
+	
+    int b0 = (key & 255);
+    int b1 = (key & 65280) >> 8;
+    int b2 = (key & 16711680) >> 16;
+    int b3 = (key & -2130706432) >> 24;
+
+    ret *= prime;
+    ret ^= b0;
+    ret *= prime;
+    ret ^= b1;
+    ret *= prime;
+    ret ^= b2;
+    ret *= prime;
+    ret ^= b3;
+
+    return ret;
+}
+
+bool 
+compareOrSet(
```

Comment:
Missed it before, but compareOrSet needs to be behind ifdef HD_HAS_PickBuffer as well.